src/components/RulerPicker.tsx

```diff
@@ -253,10 +258,19 @@ export const RulerPicker = ({
       step,
     ]
   );
+  const onContentSizeChange = useCallback(() => {
```

Comment:
@Skipperlla  Thank you! Usage of useCallback in this specific part of the component is unnecessary. Since the component's rendering is meant to occur only once, keeping useCallback here might add unnecessary overhead without providing any benefits.